### DISCUSSION TITLE
LiveCharts python not installed error when compiling WASM, but Python is installed ?

### DISCUSSION THREAD

**Discussion by digger73** [unoplatform/uno#18025] [General]
I have installed liveCharts NuGet, when compiling the WASM the error is seen as below, however as the UNO-CHECK shows Python is installed?
How to solve this ?
![image](https://github.com/user-attachments/assets/36a1b83c-d7b9-4f4b-9bce-116a168d6d6c)

**Comment by digger73** [comment]
Apparently that check doesn't seem to work very well, as it seems that I didn't have python installed. Trying to use python located there would pop up the microsoft store requesting me to download python.

**Reply by jeromelaban** [reply]
Thanks for the update. On a related note, with .NET 9, this dependency will go away, making it easier to build.
